public/opensearch.xml

```diff
@@ -0,0 +1,6 @@
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/" xmlns:moz="http://www.mozilla.org/2006/browser/search/">
+  <ShortName>IPFS Search</ShortName>
+  <Description>IPFS Search</Description>
+  <InputEncoding>UTF-8</InputEncoding>
+  <Url type="text/html" template="http://localhost:8080/#/search?q={searchTerms}" />
+</OpenSearchDescription>
```

Comment:
No trailing newline. This often leads to confusion between OS'es, particularly working with GIT.

It would be great if you could add it and/or configure your text editor to add trailing newlines.

Ref: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Thanks for the comments, I have changed the description and shortname and added the startPage parameter.